<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetAdopterProfileWindow</class>
 <widget class="QMainWindow" name="PetAdopterProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pet Adopter Profile</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout">
      <item row="0">
       <widget class="QLabel" name="labelUsername">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelUsernameValue"/>
      </item>
      <item row="1">
       <widget class="QLabel" name="labelBio">
        <property name="text">
         <string>Bio:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelBioValue"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="openImageButton">
        <property name="text">
         <string>Add Profile Picture</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="imageLabel"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEditProfile">
      <property name="text">
       <string>Edit Profile</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
